octoprint_PrintTimeGenius/analyzers/analyze_gcode_comments.py

```diff
@@ -6,7 +6,8 @@
 import sys
 from collections import defaultdict
 import argparse
-if float('.'.join((str(sys.version_info.major), str(sys.version_info.minor)))) < 3.3:
+
+if sys.version_info.major < 3 and sys.version_info.minor < 3:
```

Comment:
This is wrong.  If the version is, for example, 2.4, then it will be true and false, which is false.  That will trigger line 13.  However, that would be the wrong thing to do.

So I think that the code from before is better.  What do you think?

Reply:
of course, you are right. i will try to modify the code.